master/getting-started/kubernetes/installation/hosted/etcd.yaml

```diff
@@ -1,86 +0,0 @@
----
```

Comment:
It'd be nice to keep this file in the docs, even if we don't link to it - its used by CI.

Reply:
As discussed in Slack, let's drop this and adjust CI to use something else.